src/jsf/parser.py

```diff
@@ -126,22 +127,42 @@ def _parse(self, schema: Dict[str, Any]) -> AllTypes:
     def context(self):
         return {**self.base_context, "state": deepcopy(self.base_state)}
 
-    def generate(self, n: Optional[int] = None) -> Any:
+    def generate(self, n: Optional[int] = None, validate:Optional[bool] = False) -> Any:
         if n is None or n == 1:
+            if validate:
+                data = self.root.generate(context=self.context)
+                val(instance=data,schema=self.root_schema)
+                return data
             return self.root.generate(context=self.context)
-        return [self.root.generate(context=self.context) for _ in range(n)]
+        else:
+            data_arr = []
+            for _ in range(n):
+                if validate:
+                    data = self.root.generate(context=self.context)
+                    val(instance=data,schema=self.root_schema)
+                    data_arr.append(data)
+                else:
+                    data = self.root.generate(context=self.context)
+                    data_arr.append(data)
+            return data_arr
+
 
     def pydantic(self):
         return self.root.model(context=self.context)[0]
 
-    def generate_and_validate(self) -> None:
+    def validate(self) -> None:
         fake = self.root.generate(context=self.context)
-        validate(instance=fake, schema=self.root_schema)
+        val(instance=fake, schema=self.root_schema)
 
     def to_json(self, path: str) -> None:
         with open(path, "w") as f:
             json.dump(self.generate(), f, indent=2)
 
+    def generate_xml(self):
```

Comment:
Lets remove this to focus the package on JSON, however, it would be good to add to the docs that this is possible! Could you add this into either the README or the mkdocs?